codalab/server/rest_server.py

```diff
@@ -27,28 +26,12 @@
 
 from codalab.common import exception_to_http_error
 from codalab.lib import formatting, server_util
-import codalab.rest.account
```

Comment:
Perhaps this change is what is causing the tests to fail -- maybe you need these imports in order to add the routes?

Reply:
Hmm, let me check, somehow my IDE shows they are unused imports.

Reply:
@epicfaace I think you are right. Adding them back makes the build working. I am just curious how are these imports get called in this code...